package.json

```diff
@@ -21,7 +21,7 @@
     "url": "https://github.com/chadlieberman/graphql-type-long/issues"
   },
   "homepage": "https://github.com/chadlieberman/graphql-type-long#readme",
-  "dependencies": {
-    "graphql": "^0.9.1 || ^0.10.0"
+  "peerDependencies": {
+    "graphql": "^0.9.1 || ^0.10.0 || ^0.12.0 || ^0.13.0"
```

Comment:
I probably should have put `0.11.0` in there as well! My thinking was this:

* the project already included `0.9` and `0.10`;
* I'm using a module that has a `peerDependency` set to `0.12` and `0.13`.

So I can only really vouch for these two conditions.

Reply:
Thanks. Published on npm at version 0.1.1.